web/package.json

```diff
@@ -56,7 +56,7 @@
     "eslint": "^5.8.0",
     "eslint-plugin-vue": "^5.0.0",
     "git-revision-webpack-plugin": "^3.0.3",
-    "node-sass": "^4.11.0",
```

Comment:
Why?

Reply:
*sigh*

Because I upgraded to Node 13, which won't work with node-sass 4.11, and it's difficult to downgrade Node to a version that works with that version of node-sass.

This commit is actually in a separate PR: https://github.com/girder/viime/pull/486.

Reply:
Tested and still working with node LTS 10.16 so this is fine with me.

FWIW, https://github.com/tj/n is a thing that makes my life easier.